form/src/components/Form.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Question from "./Question";
+
+const Form = () => {
+  const handleCheckInput = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleCheckInput}>
+        <Question />
+        <button>Submit</button>
```

Comment:
why did you remove `type="submit"` attribute? It would be good to show explicitly what is the purpose of the button.